walkers/src/map.rs

```diff
@@ -13,7 +13,13 @@ use crate::{
 /// you can add it to the map with [`Map::with_plugin`]
 pub trait Plugin {
     /// Function called at each frame.
-    fn draw(&self, response: &Response, painter: Painter, projector: &Projector);
+    fn draw(
+        &self,
+        response: &Response,
+        painter: Painter,
+        projector: &Projector,
+        event_handled: bool,
```

Comment:
I think `event_handled` should go after `response`. Also, it gets interchanged with `gesture_handled`. Later name seems nicer, and it matches other things.